server/src/main/java/org/elasticsearch/ElasticsearchWrapperException.java

```diff
@@ -19,7 +19,11 @@
 
 package org.elasticsearch;
 
+/**
+ * An exception that is meant to be "unwrapped" when sent back to the user
```

Comment:
This one turned out not to be useful but I researched it so I figured I'd add the javadocs.